frontend/main.js

```diff
@@ -91,6 +91,7 @@ async function startApp () {
           errMsg: e.message || '?'
         })
       })
+      sbp('logging/flush')
```

Comment:
This is somewhat of a surprising addition to a function that is focused on displaying errors. To someone unfamiliar with the codebase, they will wonder why it is here. Could you add a comment explaining here why this is also done here?

Reply:
I included this SBP call here in an attempt to follow the suggestion you gave me on Slack, in response to one of my questions.
https://okturtles.slack.com/archives/C0EH7P20Y/p1656092332479119

Reply:
@snowteamer yeah, I'm not saying that you shouldn't have. I'm saying that an explanatory comment should go along with it on the line above to help others understand why it's here